operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-version: 4.4.0-weekly.236
-appVersion: r236
+version: 4.4.0
```

Comment:
```suggestion
version: 4.4.0-rc.0
```

Reply:
Updated in 4bc6cbd9f243b7f39e5bfcca5523f47f295fe28b